test/evergreen.yml

```diff
@@ -137,10 +137,21 @@ functions:
           echo "build --workspace_status_command=./evergreen/engflow_workspace_status.sh" >> .bazelrc.evergreen_engflow_creds
         fi
   "get project":
-    command: git.get_project
+  - command: git.get_project
     type: setup
     params:
       directory: wiredtiger
+  - command: shell.exec
+    type: setup
+    params:
+      shell: bash
+      working_dir: wiredtiger
+      script: |
+        set -o errexit
+        set -o verbose
+        if [ "$OS" != "Windows_NT" ] && ! [ -d "./automation-scripts" ]; then
```

Comment:
@lukech I suspect this is likely not needed, but I don't know enough about the build env to be sure.

Reply:
It depends on where the function (where this `shell.exec` command resides) will be called. For this case, I believe it's safe to assume we will not run tcmalloc builds on Windows and the folder does not exist in an Evergreen task run.

Reply:
Looks like it's working without the OS check, I kept the directory check in because it seem some builds call compile_wiredtiger more than once.